diff --git a/node_modules/primevue/inputnumber/InputNumber.vue b/node_modules/primevue/inputnumber/InputNumber.vue
index 2f06e7c..4d62fec 100644
--- a/node_modules/primevue/inputnumber/InputNumber.vue
+++ b/node_modules/primevue/inputnumber/InputNumber.vue
@@ -358,7 +358,6 @@ export default {
         },
         onUpButtonMouseDown(event) {
             if (!this.disabled) {
-                this.$refs.input.$el.focus();
                 this.repeat(event, null, 1);
                 event.preventDefault();
             }
@@ -385,7 +384,6 @@ export default {
         },
         onDownButtonMouseDown(event) {
             if (!this.disabled) {
-                this.$refs.input.$el.focus();
                 this.repeat(event, null, -1);
                 event.preventDefault();
             }
diff --git a/node_modules/primevue/inputnumber/index.mjs b/node_modules/primevue/inputnumber/index.mjs
index 9885ab7..853b875 100644
--- a/node_modules/primevue/inputnumber/index.mjs
+++ b/node_modules/primevue/inputnumber/index.mjs
@@ -428,7 +428,6 @@ var script = {
     },
     onUpButtonMouseDown: function onUpButtonMouseDown(event) {
       if (!this.disabled) {
-        this.$refs.input.$el.focus();
         this.repeat(event, null, 1);
         event.preventDefault();
       }
@@ -455,7 +454,6 @@ var script = {
     },
     onDownButtonMouseDown: function onDownButtonMouseDown(event) {
       if (!this.disabled) {
-        this.$refs.input.$el.focus();
         this.repeat(event, null, -1);
         event.preventDefault();
       }
